torch/_inductor/codegen/wrapper.py

```diff
@@ -1711,8 +1719,12 @@ def fill_args(arg, arg_type):
 
         def fill_output_arg(arg, return_type):
             if isinstance(return_type, torch.TensorType):
-                self.writeline(f"at::Tensor {arg};  // output buffer")
-                new_tensor_args.append(f"&{output_arg}")
+                self.writeline(f"AtenTensorHandle {arg}_handle;  // output buffer")
+                self.writeline(
+                    f"AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_new_uninitialized_tensor(&{arg}_handle));"
+                )
+                self.writeline(f"RAIIAtenTensorHandle {arg}({arg}_handle);")
```

Comment:
Hmm, wondering if we need to guard these lines with ```config.aot_inductor.abi_compatible```?

Reply:
no, `generate_extern_kernel_args_decl_if_needed` is only used in the fbcode path, which always need abi=true.